pkg/distributors/test/DistributionScheduler.test.ts

```diff
@@ -207,34 +220,109 @@ describe('Distribution Scheduler', () => {
       });
     });
 
-    // Skip tests pending support for paying into a distribution owned by another address
-    context.skip('when distribution has been started', () => {
+    context('when distribution has already been started', () => {
       let scheduleId: string;
       sharedBeforeEach(async () => {
         const distributionStartTime = await fromNow(DAY);
         await scheduler
           .connect(distributionOwner)
-          .scheduleDistribution(
-            distributionId,
-            stakingToken.address,
-            distributionToken.address,
-            amount,
-            distributionStartTime
-          );
+          .scheduleDistribution(distributionId, DISTRIBUTION_AMOUNT, distributionStartTime);
 
-        scheduleId = await scheduler.getScheduleId(
-          stakingToken.address,
-          distributionToken.address,
-          distributionOwner.address,
-          distributionStartTime
-        );
-        await advanceTime(3600 * 25);
+        scheduleId = await scheduler.getScheduleId(distributionId, distributionStartTime);
+        await advanceTime(DAY + HOUR);
         await scheduler.connect(other).startDistributions([scheduleId]);
       });
 
+      it('skips the distribution', async () => {
+        const receipt = await (await scheduler.connect(other).startDistributions([scheduleId])).wait();
+
+        await expectEvent.notEmitted(receipt, 'DistributionStarted');
+      });
+    });
+  });
+
+  describe('cancelDistribution', () => {
```

Comment:
Let's add a test where we check starting a cancelled distribution does nothing (it is skipped).